README.md

```diff
@@ -1,15 +1,15 @@
 # discourse-mermaid-theme
 
-```
-[wrap=mermaid height=500]
-gantt
-    title A Gantt Diagram
-    dateFormat  YYYY-MM-DD
-    section Section
-    A task           :a1, 2019-01-01, 30d
-    Another task     :after a1  , 20d
-    section Another
-    Task in sec      :2019-01-12  , 12d
-    another task      : 24d
-[/wrap]
-```
+````
+    ```mermaid height=500
+    gantt
+        title A Gantt Diagram
+        dateFormat  YYYY-MM-DD
+        section Section
+        A task           :a1, 2019-01-01, 30d
+        Another task     :after a1  , 20d
+        section Another
+        Task in sec      :2019-01-12  , 12d
+        another task      : 24d
+    ```
```

Comment:
Is there a reason for the indentation?

Reply:
I don't think it works otherwise:


```
```test
foo
```
```